package.json

```diff
@@ -20,6 +20,7 @@
     "build": "tsc",
     "watch": "tsc --watch",
     "test": "nyc mocha",
+    "test-fluent": "cd fluent; npm test",
```

Comment:
I would suggest putting this inside the same `test` param as then you don't need to update the readme's / ci / docs.

Also, you might want to do `"test": "nyc mocha && cd fluent && npm test"` this way it will still fail if it can't `cd` or run `mocha`